### PR TITLE
Show error message on failed login

### PR DESCRIPTION
**Pull request by elijahkim**

### CODE REVIEW

File: app/components/Errors.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { map } from "lodash";
+import CSSModules from "react-css-modules";
+import styles from "../styles/components/errors.css";
+
+class Errors extends React.Component {
+  render () {
+    const { errors } = this.props;
+    const errorsMap = map(errors, (error, index) => {
```
Comment (cjbell) [inline app/components/Errors.js]: Prefer doing this inline inside the JSX rather than assigning to a var

File: app/components/Errors.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      return <li key={index} styleName="list-item">{error}</li>
+    });
+
+    return (
+      <ul styleName="unordered-list">{errorsMap}</ul>
```
Comment (cjbell) [inline app/components/Errors.js]: This is a bad class name, try to make something more descriptive like `errors` or `errors-list`

File: app/components/Errors.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { map } from "lodash";
+import CSSModules from "react-css-modules";
+import styles from "../styles/components/errors.css";
+
+class Errors extends React.Component {
+  render () {
+    const { errors } = this.props;
+    const errorsMap = map(errors, (error, index) => {
+      return <li key={index} styleName="list-item">{error}</li>
```
Comment (cjbell) [inline app/components/Errors.js]: Same here – should be something like `error` or `error-msg`

File: app/components/Errors.js (hunk 1 above)
Comment (elijahkim) [inline app/components/Errors.js]: can you explain why?

Comment (cjbell) [inline app/components/Errors.js]: No need for intermediate variable assignment:

``` js
return (
  <ul styleName="unordered-list">
   {map(errors, (error, index) => {
       return (
         <li key={index} styleName="list-item">
          {error}
         </li>
       )
   })
  </ul>
)
```

Comment (elijahkim) [inline app/components/Errors.js]: ehhhhhhh I'm not sold. Seeing too much js mixed with mark up seems a little off to me. Also the const kind of describes whats going on. What do you think of creating a property `renderErrors(errors)` or something instead?

Comment (cjbell) [inline app/components/Errors.js]: But it's not markup, it's just JS once it's compiled. That's the beauty in JSX hence why I think this kind of thing is OK. 

I think for simple loops like this, it's fine. Anything more in the child and you should break out to another function.

**Pull request merged**